docs/source/Installation.rst

```diff
@@ -302,50 +275,117 @@ From the conda environment you used to install CaImAn:
-on your own data to save them as a different file to avoid losing your work when updating the caiman_data directory.
+   <details>
+   <summary>Upgrade the demos</summary>
+
+When you upgrade Caiman, sometimes the underlying APIs change. When this happens and it impacts a demo (or otherwise 
+requires changes to files in ``caiman_data``), we update the demo and data. This means that upgrading CaImAn works 
+best if you also replace the ``caiman_data`` directory with a new version.
+
+However, you may have made your own changes to the demos (e.g. to work with your data). If you have done this, 
+you may need to massage your changes into the new versions of the demos. For this reason, we recommend that if 
+you modify the demos to operate on your own data to save them as a different file to avoid losing your work 
+when updating the caiman_data directory.
 
-To update the caiman_data directory you can follow the following procedure:
+To update ``caiman_data`` you can follow the following procedure:
 
 - If there are no new demos or files in the new CaImAn distribution, then you can leave it as is.
 
-- If you have not modified anything in caiman_data but there are upstream changes in the new CaImAn distribution, then remove caiman_data directory before upgrading and have caimanmanager make a new one after the upgrade.
+- If you have not modified anything in caiman_data but there have been changes in the new Caiman release, 
+  then remove ``caiman_data`` directory before upgrading and have ``caimanmanager`` make a new one after the upgrade, by 
+  running caimanmanager as discussed in Section 2.
+
+- If you have extensively modified things in ``caiman_data``, rename your ``caiman_data`` directory, and have caimanmanager 
+  make a new one after the upgrade, and then massage your changes back in. E.g., if you have extensively 
+  modified ``demo_pipeline.ipynb`` for your personal use-case, then change the name of this notebook before putting it back into ``caiman_data``.
+
+.. raw:: html
+
+   </details>
+
+Section 4: Miscellaneous
+-------------------------
+
+Section 4A: Setting up environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- If you have extensively modified things in caiman_data, rename the caiman_manager directory, have caimanmanager make a new one after the upgrade, and then massage your changes back in.
+To make the package work *efficiently* and eliminate "crosstalk" between
+different processes, some multithreading operations need to be turned off
+This is for Linux and Windows and is not necessary is OSX. This process is
+not needed if you installed using conda/mamba.
 
+For **Linux (and OSX)** run these commands before launching Python:
+
+.. code:: bash
 
-Installing additional packages
-------------------------------
+     export MKL_NUM_THREADS=1
```

Comment:
setting these env vars is only important for dev installs now; the pure-conda installs set this via activate scripts.

Reply:
Oh, we already note this. But maybe let's hide this away then in a separate section for compile-based installs; most users shouldn't need to see this.

Reply:
I added a note about this in the original dev-mode install section.